### PR TITLE
refactor counter tests

### PR DESCRIPTION
**Pull request by xcoulon**
use func to specify all the base values, including
the number of master user records.

this change will allow for specifying more values/metrics
in the future ;)

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: pkg/counter/cache.go [hunk 1]
```diff
@@ -16,14 +17,20 @@ var log = logf.Log.WithName("counter_cache")
...
 }
 
 // Counts is type that contains number of MURs and number of UserAccounts per member cluster
 type Counts struct {
-	MasterUserRecordCount        int
+	// Total number of Master User Accounts
+	MasterUserRecordCount int
+	// User Accounts per Clusters (indexed by cluster name)
 	UserAccountsPerClusterCounts map[string]int
+	// Activations per Users (number of users indexed by their number of activations)
```
Comment (alexeykazakov) [inline pkg/counter/cache.go:32]: It's not activations per user. It's number of users per activation count. Correct?

**Review by alexeykazakov — APPROVED** [review]
Nice improvement. But it's not only about tests, is it?

File: pkg/counter/cache.go (hunk 1 above)
Comment (xcoulon) [reply pkg/counter/cache.go:32]: ah right, I cherry-picked a change to make a separate PR and:
- I did not realize this file was affected in this PR (it should not)
- I forgot to edit the commend (the initial var name was misleading, as you noted)

Comment (xcoulon) [reply pkg/counter/cache.go]: removed in f9a2b08

**Pull request merged**